packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,286 @@
+            onClick={this._onClickSeeMore}
+            onRenderText={this._onRenderText}
+            styles={{ root: { paddingLeft: '0px' }, textContainer: { paddingLeft: '4px' } }}
+          >
+            See More
+          </ActionButton>
+          {this.state.showSeeMore && (
+            <Stack gap={LARGE_GAP_SIZE}>
+              {this.state.properties.map((item: IApiProperty, index: number) =>
+                index !== 0 ? (
+                  <ApiReferencesTable
+                    key={item.propertyName}
+                    name={item.propertyName}
+                    title={item.title}
+                    description={item.description}
+                    extendsTokens={item.extendsTokens}
+                    properties={item.property}
+                    methods={item.methods}
+                    renderAsEnum={item.propertyType === PropertyType.enum}
+                    renderAsClass={item.propertyType === PropertyType.class}
+                  />
+                ) : (
+                  undefined
+                )
+              )}
+            </Stack>
+          )}
+        </Stack>
+      );
+    }
+    return undefined;
+  }
+
+  public render(): JSX.Element {
+    const { className } = this.props;
+    return (
+      <Stack gap={LARGE_GAP_SIZE} className={css(apiReferencesTableTopMargin, className)}>
+        {this.renderFirst()}
+        {this.renderEach()}
+      </Stack>
+    );
+  }
+
+  public componentDidMount(): void {
+    window.addEventListener('hashchange', this._onHashChange);
+  }
+
+  public componentWillUnmount(): void {
+    window.removeEventListener('hashchange', this._onHashChange);
+  }
+
+  public componentDidUpdate(prevProps: IApiReferencesTableSetProps, prevState: IApiReferencesTableSetState): void {
+    if (prevState.showSeeMore === false && this.state.showSeeMore === true) {
+      const hash = this._extractAnchorLink(window.location.hash);
+      const el = document.getElementById(hash);
+
+      if (el) {
+        // update scroll position
+        window.scrollTo({ top: el.offsetTop - 31.5 /* title */, behavior: 'smooth' });
+      }
+    }
+  }
+
+  private _onHashChange = (): void => {
+    const { properties, showSeeMore } = this.state;
+
+    const hash = this._extractAnchorLink(window.location.hash);
+    const el = document.getElementById(hash);
+
+    if (el) {
+      // update scroll position
+      window.scrollTo({ top: el.offsetTop - 31.5 /* title */, behavior: 'smooth' });
+    }
+
+    if (!showSeeMore) {
+      const section = properties.filter(x => x.propertyName === hash)[0];
+      if (section) {
+        this.setState({
+          showSeeMore: true
+        });
+      }
+    }
+  };
+
+  private _extractAnchorLink = (path: string): string => {
+    const split = path.split('#');
+    const cleanedSplit = split.filter(value => !!value);
+    return cleanedSplit[cleanedSplit.length - 1];
```

Comment:
Consider re-using what you had defined previously, or is the difficulty that it is in a separate package?